schemas/config/merge_replicates.schema.yaml

```diff
@@ -0,0 +1,11 @@
+$schema: "http://json-schema.org/draft-06/schema#"
+
+description: snakemake-workflows replciate merging configuration
+
+properties:
+  # replicate handling
+  combine_replicates:
+    description: which method to use to combine replicates (concatenation, idr or fisher)
+    default: merge # for technical replicates
+    enum: [fisher, idr, merge]
+    type: string
```

Comment:
Why not add this to alighment_general schema?

Reply:
can do, but then merge_replicates.smk should also be rolled into alignment.smk. Furthermore, this rule will be expanded upon when we allow for multiple types of merging, as per #186.